swap_meet/vendor.py

```diff
@@ -1,2 +1,49 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+    def get_by_category(self, category):
+        category_items = []
+        for item in self.inventory:
+            if item.category == category:
+                category_items.append(item)
+        return category_items
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item in self.inventory and \
+            their_item in vendor.inventory:
+            self.remove(my_item)
+            self.add(their_item)
+            vendor.remove(their_item)
+            vendor.add(my_item)
+            return True
+    
+    def swap_first_item(self, vendor):
+        if self.inventory and vendor.inventory:
+            return self.swap_items(vendor, self.inventory[0], vendor.inventory[0])
```

Comment:
great use of swap_items! Don't forget to return False here as well.

Reply:
I thought that and empty list will be Falsy and I could skip the len check ?

Reply:
You definitely should use a check but it doesn't have to use len(). Another suggestion would be `if self.inventory and vendor.inventory:`